test/e2e/framework/kubelet/stats.go

```diff
@@ -551,11 +551,11 @@ func (r *ResourceMonitor) GetLatest() (ResourceUsagePerNode, error) {
 // GetMasterNodeLatest returns the latest resource usage of master and node.
 func (r *ResourceMonitor) GetMasterNodeLatest(usagePerNode ResourceUsagePerNode) ResourceUsagePerNode {
```

Comment:
Why don't you rename this function?

Reply:
I wasn't sure where this function was being referenced so I decided to not touch it until I'm sure.

Reply:
The function is called from https://github.com/kubernetes/kubernetes/blob/b47f8263e18c7b13dba33fba23187e5e0477cdbd/test/e2e/node/kubelet_perf.go#L112 only according to https://cs.k8s.io/?q=GetMasterNodeLatest&i=nope&files=&excludeFiles=&repos=